### ISSUE TITLE
Cannot read property 'AlwaysExpand' of null

### ISSUE THREAD

**Issue by Triloworld** [phpmyadmin/phpmyadmin#17620]
Latest version of Docker Image or any from 4.9 start.
Can be related to: https://github.com/phpmyadmin/phpmyadmin/issues/16428
Opened as this comment suggest: https://github.com/phpmyadmin/phpmyadmin/issues/16428#issuecomment-1141297273

Reproduce:
if env variable isn't proper it will mess and show popup and error like in title of this issue:
I set by mistake on Compose image in stack:
 - PMA_ABSOLUTE_URI=https://`${domain}`/

After commenting error gone.

Solution:
Inform that env variable isn't proper rather than cryptic message. 

Fast patch:
Add catch and notify that issue can be in PMA_ABSOLUTE env and require to be manually verified

**Comment by williamdes** [comment]
Hi @Triloworld
Can you provide your docker-compose file?

**Comment by Triloworld** [comment]
```
version: '3.9'
services:

  wp_phpmyadmin:
    image: phpmyadmin/phpmyadmin:latest
    environment:
      - PMA_ARBITRARY=1
      - PMA_HOSTS=wordpress_db
      - PMA_PORT=3306
      #- PMA_USER=user
      #- PMA_PASSWORD=password
      - PMA_ABSOLUTE_URI=https://db.`${domain}`/
      - HIDE_PHP_VERSION=true
      - UPLOAD_LIMIT=200M
    ports:
      - 9090:80
    depends_on:
      - wordpress_db
    networks:
      - backend
```

If I ommit env for `domain` then url is "https://db./" It happen because my env aren't setup when I create service. This lead to error above as domain IS input but is incorrect with used one

**Comment by williamdes** [comment]
Thank you, by the way you should use the Docker official repository
```diff
- phpmyadmin/phpmyadmin:latest
+ phpmyadmin:latest
```
See: https://github.com/phpmyadmin/docker/issues/363

**Comment by Triloworld** [comment]
Then pleas, add any notification in console about deprecation. Error is on any version regarding source. Can you verify is that correct? is error show up?

**Comment by williamdes** [comment]
> Then pleas, add any notification in console about deprecation. Error is on any version regarding source. 

This error is not expected and need to be fixed :) 


> Can you verify is that correct? is error show up?

I will try as soon as possible :)

**Comment by rbozan** [comment]
I have the same using the following:

```
version: "3.9"
services:
  db:
    image: mariadb:10.3
    environment:
      MYSQL_ROOT_PASSWORD: notSecureChangeMe

  phpmyadmin:
    image: phpmyadmin:5.2-apache
      #phpmyadmin:5.1-fpm-alpine
    # build: .
    depends_on:
      - db
    ports:
      - 80:80
    environment:
      # - PMA_ARBITRARY=10.3
      PMA_USER: root
      PMA_PASSWORD: notSecureChangeMe

      # See config.user.inc.php
      PMA_USER_CONFIG_BASE64: PD9waHANCiRjZmdbJ0FsbG93VGhpcmRQYXJ0eUZyYW1pbmcnXSA9IHRydWU7DQokY2ZnWydWZXJzaW9uQ2hlY2snXSA9IGZhbHNlOw==
```

However, this only happens for me when I have the following code:

```html
<p><iframe src="http://127.0.0.1:80" width="400px" height="1000px">
  </iframe></p>
```

What I mean by this is when I visit the URL in my browser like so: http://127.0.0.1:80 there does not seem to be any problem, but it only appears when I have it in an `iframe`.

Left is using a normal browser tab (with some Chrome extensions), right is using the `iframe`.
![image](https://user-images.githubusercontent.com/7997154/181103198-a9e16931-4d57-43a7-88e2-e6378775acfa.png)

**Comment by williamdes** [comment]
Thank you @rbozan !
I can confirm the issue, it's because the cookies are not sent to the server because of the SameSite security.
I am working on a fix for the JS part and maybe more fixes

**Comment by williamdes** [comment]
I pushed a9356e4d16579103292a0ec03ffb39373ffaafe8, could you apply it locally in the container and let me know if it works for you too ?

Meanwhile I am searching why only the GET request hits success and not the POST request to get console data
When it is solved maybe the console HTTP request will have a other response than now
```
curl 'http://127.0.0.1:8095/index.php?route=/server/status&lang=en&ajax_request=true&ajax_page_request=true&_nocache=166056380073132233&token=793b555f314d5f41786b2f337167472c'

curl 'http://127.0.0.1:8095/index.php?route=/config/get' \
  -H 'Content-Type: application/x-www-form-urlencoded; charset=UTF-8' \
  --data-raw 'ajax_request=true&server=1&key=NavigationWidth&_nocache=1660563550733690164&token=793b555f314d5f41786b2f337167472c'
```

**Comment by williamdes** [comment]
Is this now fixed with 5.2.1 in Docker ?
